pymel/core/__init__.py

```diff
@@ -18,7 +17,6 @@
 import pymel.internal.pmcmds as _pmcmds
 _pmcmds.addAllWrappedCmds()
 
-import pymel.api as _api
```

Comment:
I changed this import from `_api` to `api` because we were just assigning this to `api` below